controllers/universitiesController.ts

```diff
@@ -0,0 +1,42 @@
+import { Request, Response } from 'express';
+import { param } from 'express-validator';
+import dbUniversities from '../db/universities';
+import asyncHelper from '../utils/async-helper';
+import validation from '../utils/validation';
+import CustomError from '../utils/CustomError';
+
+const getAll = async (req: Request, res: Response): Promise<void> => {
+  const universities = await dbUniversities.getAllOrderByName();
```

Comment:
There was an extra sort here, I removed it, we are already sorting in the request.